crates/solidity/testing/snapshots/cst_output/YulStatement/var_assign_colon_and_equals/input.sol

```diff
@@ -0,0 +1 @@
+newB: = mload(0x80)
```

Comment:
I suggest adding a comment to explain the reason, so that we don't miss it in any future grammar update:

```suggestion
// Make sure that `newB:` is not parsed as a label here, and that we continue parsing to get a whole assignment statement..
newB: = mload(0x80)
```